Automattic/wp-calypso
# Add email verified notification to launchpad

#### Proposed Changes

* Adds `global-notices` component to stepper flow and adds specific logic to handle`Email Confirmed!` global notice on Launchpad page if URL contains `verified` query param.

#### Testing Instructions



* Checkout this branch and run `yarn start`.
* Create a new newsletter or link-in-bio site: https://wordpress.com/hp-2022-tailored-flows/
* Go through the site creation flow until you reach the Launchpad screen.
* Navigate to: `calypso.localhost:3000/home/{SITE_SLUG}?verified=1` to simulate an email confirmation notification. You should get redirected to the Launchpad and you see an `Email confirmed!` notification at the top of the Launchpad page.

<img width="699" alt="image" src="https://user-images.githubusercontent.com/10482592/193146325-242abe1c-81ea-4468-a948-b2a6c3b3ecc0.png">


#### Pre-merge Checklist






Related to #68407
